.github/workflows/codeql-analysis.yml

```diff
@@ -20,8 +20,15 @@ on:
   schedule:
     - cron: '35 17 * * 2'
 
+permissions:
```

Comment:
Since we only have one job, I don't think we need this top level key, right?

Reply:
Yes, removed it. Thanks